tests/unit/test_metadata_splitter.py

```diff
@@ -226,6 +233,7 @@ def test_splitting():
         timestamp=timestamp,
         configs=configs,
         metrics=metrics,
+        string_series=None,
```

Comment:
I'd very much like to have this tested. Not sure if it should be the same test case (but possibly).